### DISCUSSION TITLE
What do you use for a hamburger menu ?

### DISCUSSION THREAD

**Discussion by peterDijk** [nuxt/nuxt#10865] [Questions (Nuxt 2)]
Hi, 

I have a hard time finding a solution for a hamburger menu that works. 
So, what do you use for a hamburger menu ? 

Thanks!
Peter

**Comment by hslee2008** [comment]
Vuetify Navigation Drawer and the hamburger menu works fine for me.
![Screenshot 2023-01-17 145220](https://user-images.githubusercontent.com/78584173/212820254-804360cd-6b69-4b07-b24b-9ff4ad8c0e1d.png)

https://vuetifyjs.com/en/components/app-bars/#usage

**Comment by ojvribeiro** [comment]
Before the frameworks hype, I used to use this:

https://jonsuh.com/hamburgers/

**Comment by Baroshem** [comment]
I would not recommend to use vuetify library just to have one component. The usage of component library should be a global decision for a project and used everywhere. (Believe me, in my current project we are now migrating from Vuetify to TailwindCSS for similar reason).

There is a cool tutorial on how to build a Hamburger icon with Vue and Tailwind -> https://larainfo.com/blogs/vue-3-responsive-navbar-menu-with-tailwind-css-example
